modules/ROOT/pages/addition/docker/container.adoc

```diff
@@ -1,4 +1,4 @@
-= Monitor a Neo4j instance in a standalone Docker container
+= Monitor Neo4j instance in standalone Docker
```

Comment:
```suggestion
= Monitor instance in standalone Docker container   
```
I think we need to have _container_ in there. This is only slightly longer, I don't know what the guidelines are? Will it work?

Reply:
No, it's still ok! It needs to be up to 70 characters including the main page name... so it would be "Monitor instance in standalone Docker container - Neo4j Ops manager" - 67 characters!